Pargi/MapViewController.swift

```diff
@@ -130,11 +145,17 @@ class MapViewController: UIViewController, MKMapViewDelegate, UIGestureRecognize
         self.mapView.setCamera(camera, animated: false)
         
         self.previousBottomDistance = distance + bottomSafeArea
+        
+        // Update user location centering button coordinates based on drawer height
+        locateUserButtonBottomConstraint.constant = distance * -1
```

Comment:
Super :neckbeard:, but simply `-distance` would look nicer (and be the smallest bit faster).

Reply:
Ah yes, of course. I meant to fix that, but forgot.